db/migrate/20230709181609_drop_archived_container_projects.rb

```diff
@@ -0,0 +1,12 @@
+class DropArchivedContainerProjects < ActiveRecord::Migration[6.1]
+  class ContainerProject < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    ContainerProject.where.not(:old_container_group_id => nil)
+                    .where(:container_group_id => nil)
+                    .update_all('container_group_id = old_container_group_id')
```

Comment:
I think you want `ContainerGroup.where_not(:old_container_project_id => nil)...etc`